test/Project.toml

```
@@ -9,6 +9,7 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
 Aqua = "0.7, 0.8"
+ExplicitImports = "1"
```

```suggestion
ExplicitImports = "1.0.1"
```
We need at least v1.0.1 because before Julia 1.9 was not supported, which is used in the tests.